lua/snacks/words.lua

```diff
@@ -18,7 +18,9 @@ local defaults = {
   foldopen = true, -- open folds after jumping
   jumplist = true, -- set jump point before jumping
   modes = { "n", "i", "c" }, -- modes to show references
-  exclude_ft = {}, -- filetypes to exclude
+  filter = function(buf) -- what buffers to enable `snacks.words`
+    return vim.g.words_disable ~= false and vim.b[buf].words_disable ~= false
```

Comment:
can you change this to `vim.g.snacks_words ~= false ...`?
We use the same naming scheme in other places

Reply:
Yes, done. Sorry I didn't think about naming scheme.